src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/planner/converter/SQLAggregationParserTest.java

```diff
@@ -247,7 +306,28 @@ public void aggregationWithNestedShouldThrowException() {
                      + "FROM t "
                      + "GROUP BY nested(projects.name.keyword, 'projects')";
         SQLAggregationParser parser = new SQLAggregationParser(new ColumnTypeProvider());
-        parser.parse(mYSqlSelectQueryBlock(sql));
+        MySqlSelectQueryBlock query = mYSqlSelectQueryBlock(sql);
+        List<String> selectMethodNames = prepareRawMethodNames(query, true);
+        parser.parse(query, selectMethodNames);
+    }
+
+    private List<String> prepareRawMethodNames(MySqlSelectQueryBlock query, Boolean fillWithNull) {
```

Comment:
Is it the same function used in SQLToOperatorConverter?

Reply:
Same logic. Might be duplicate code with the intent to keep parserTest separate from converter.